lib/active_model/serializer/associations.rb

```diff
@@ -82,14 +82,17 @@ def associate(reflection)
         end
       end
 
+      # @param [IncludeTree] include_tree (optional)
       # @return [Enumerator<Association>]
       #
-      def associations
+      def associations(include_tree = nil)
         return unless object
 
+        include_tree ||= ActiveModel::Serializer::IncludeTree.from_string('*')
```

Comment:
Actually, I'm not sure it's the best solution, since it would expose the `self.default_include_tree` method to the public. Maybe make it a constant member of the class? We don't event need to `dup` actually.